### PR TITLE
Update plot_cm.py

### PR DESCRIPTION
**Pull request by seblome**
- Added network-specific coloring of the CM border when specifying cm_bordercolor = 'networks'.
- Removed the possibility for alpha to vary between different networks when drawing boundaries.
- Added a new kwarg "cm_boundaryalpha" that adjusts alpha of the CM boundary.

### CODE REVIEW

**Comment by wiheto** [comment]
Some comments: 

- [ ] could you add documentatoin about cm_bordercolor and cm_boundaryalpha to docs/api.md. Should be straight forward. 
- [ ] could you add a default value (1, I guess) of cm_boundaryalpha to netplotbrain/profiles/default.json

After that I will update the tests and add a test for this code as well.

**Comment by wiheto** [comment]
Ignore my pull request to your branch. But if you could add these small changes to the documentation we can merge this.

File: netplotbrain/plotting/plot_cm.py [hunk 1]
```diff
@@ -69,15 +73,39 @@ def _plot_connectivitymatrix(ax, edges, nodes=None, node_color=None, node_colorb
...
-        ax.plot([number_of_nodes, 0], [number_of_nodes, number_of_nodes], color=cm_bordercolor, linewidth=cm_borderwidth, transform=tr,zorder=5)
-        ax.plot([0, number_of_nodes], [0, 0], color=cm_bordercolor, linewidth=cm_borderwidth, transform=tr,zorder=5)
-        ax.plot([number_of_nodes, number_of_nodes], [0, number_of_nodes], color=cm_bordercolor, linewidth=cm_borderwidth, transform=tr,zorder=5)
+    if cm_border:
+        if cm_bordercolor == 'networks':
+            line_blank = np.zeros(number_of_nodes+1)
+            line_fill = np.zeros(number_of_nodes+1)
+            for i in range(number_of_nodes+1):
+                line_fill[i] = i-0.5
+            points_x = np.array([line_fill, line_blank]).T.reshape(-1, 1, 2)
```
[the remark this answers is not in the document]
Comment (seblome) [reply netplotbrain/plotting/plot_cm.py]: I added the extra parenthesis just now, thanks!

**Comment by wiheto** [comment]
Super! LGTM

**Pull request merged**